skl2onnx/common/_topology.py

```diff
@@ -66,7 +66,7 @@ def __init__(self, raw_name, onnx_name, scope, type=None):
                     raise TypeError("shape must be a tuple or a list not "
                                     "{}.".format(type_fct(shape)))
             for dim in shape:
-                if dim == 'None':
+                if dim in ('None', 'N', None):
```

Comment:
What do 'None' and 'N' mean?

Reply:
'N' is not needed anymore, I'll revert it. Wenbing made a change in onnxconverter-common, and I replaced 'N' by None everywhere and this means one dimension is unknown.